### ISSUE TITLE
`locale` option not working

### ISSUE THREAD

**Issue by c4ffein** [antoniandre/vue-cal-v4#353]
Hi,

I met a bug using the `locale` option, the words don't load and I have this error in my console:

```vuecal.common.js?7fa7:107 Uncaught (in promise) ChunkLoadError: Loading chunk 11 failed.
(missing: http://localhost:8080/js/vuecal.common.i18n/fr.js)
    at requireEnsure (webpack-internal:///./node_modules/vue-cal/dist/vuecal.common.js:107:26)
    at Array.map (<anonymous>)
    at webpackAsyncContext (webpack-internal:///./node_modules/vue-cal/dist/vuecal.common.js:1595:34)
    at VueComponent.loadLocale (webpack-internal:///./node_modules/vue-cal/dist/vuecal.common.js:8939:36)
    at VueComponent.created (webpack-internal:///./node_modules/vue-cal/dist/vuecal.common.js:9839:10)
    at invokeWithErrorHandling (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:1853:57)
    at callHook (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:4213:7)
    at VueComponent.Vue._init (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:4998:5)
    at new VueComponent (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5144:12)
    at createComponentInstanceForVnode (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:3280:10)```

Could it be linked with the fact that https://github.com/antoniandre/vue-cal/blob/5831735c00462eceff2799296bcb9a17b2d6c8d0/src/vue-cal/index.vue#L365
has been modified to fix a babel bug, and due to operator precedence the Webpack module method doesn't work as expected?
If not, could you confirm that the locale option is still working with a recent Vue v2 setup?

Thanks in advance.

**Comment by antoniandre** [comment]
Bonjour @c4ffein,
I can confirm the locale is still working with the latest vue-cal on Vue 2. 
You can check this demo: https://codepen.io/antoniandre/pen/yLJWBbK?editors=1010

It is also working on the documentation page which is Vue 2.
https://antoniandre.github.io/vue-cal/#ex--internationalization

How did you load the locale? A correct path should be `vue-cal/dist/i18n/fr.js`.
Cheers.

**Comment by antoniandre** [comment]
By the way I just released version 3.9.0 where you can provide a fully custom locale. 
Not what I recommend, but if in your project you have no other choice, then here it is: #352.

**Comment by c4ffein** [comment]
Hi again,

I just realized I made the import in the wrong file. Sorry for wasting your time and thanks for your support.
